mezzanine/shiratech-cam1/README.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Documentation Shiratech Camera 1 Mezzanine
+permalink: /documentation/mezzanine/shiratech-cam1/
+---
+
+## Shiratech Camera 1
+
+# Table of Contents
+- [1) Introduction](#1-introduction)
+  - [1.1) Specifications & Features](#11-specifications-and-features)
+- [2) Guides and Documentation](#2-guides-and-documentation)
+
+# 1) Introduction
+
+<img src="https://github.com/96boards/website/blob/master/_product/mezzanine/shiratech-cam1/images/shiratech-cam1-front-sd.png?raw=true" data-canonical-src="https://github.com/96boards/website/blob/master/_product/mezzanine/shiratech-cam1/images/shiratech-cam1-front-sd.png?raw=true" width="300" height="250" />
```

Comment:
Please do not reference images stored on GitHub. Please make these links relative to the 96Boards website.

Reply:
that's because the website PR for this project hasn't been merged and built. this is just a placeholder till that happens.